packages/hooks/src/useCashierLocked.ts

```diff
@@ -0,0 +1,13 @@
+import { useStore } from '@deriv/stores';
+
+const useCashierLocked = () => {
+    const { client } = useStore();
+    const { account_status } = client;
+
+    const is_cashier_locked =
+        !!account_status?.status && account_status.status.some(status_name => status_name === 'cashier_locked');
```

Comment:
```suggestion
        account_status.status.some(status => status === 'cashier_locked');
```
@rostislav-deriv I don’t think we need to check if `account_status` is null or not as the type says it cannot be, unless the type we have in wrong 🤔

Reply:
Seems like this change produces this error, so we need the extra check

<img width="1419" alt="Screenshot 2023-03-07 at 11 30 20" src="https://user-images.githubusercontent.com/119863957/223366930-30db9112-f5da-42a6-9f66-4641984610da.png">

Reply:
The extra check in question would be the same as in `useIsSystemMaintenance`